crates/pyrefly_types/src/display.rs

```diff
@@ -87,6 +89,7 @@ pub struct TypeDisplayContext<'a> {
     /// Should we display for IDE Hover? This makes type names more readable but less precise.
     hover: bool,
     always_display_module_name: bool,
+    display_modules: RefCell<SmallSet<ModuleName>>,
```

Comment:
```suggestion
    modules: RefCell<SmallSet<ModuleName>>,
```
maybe we don't need the display_ prefix here since it's already part of the displaycontext struct. a docstring with more information would be nice